app/scss/questionPage.scss

```diff
@@ -1,3 +1,4 @@
+@import "imports/variables";
```

Comment:
Is it really necessary to import a file that has 2 variables in it? Will be variables used elsewhere or only in the question page? If they're only used here I don't see why they shouldn't just be included at the top of this file.

Reply:
There will be more vars in future stories - doing this now to avoid technical debt.